docs/src/design/stack/field_ops.md

```diff
@@ -208,7 +214,7 @@ The `acc` in the next frame is the product of `val` and `acc` in the current fra
 s_2' - s_2 * h_0 = 0 \text{ | degree} = 2
 $$
 
```

Comment:
some of the variable names in the remaining constraints should be updated as well

Reply:
I left those as-is, since this is the convention that the rest of the file adopts (i.e. use "real names" in the diagram, but then use "stack indices" in the constraints description).